code-challenges.test.js

```diff
@@ -14,33 +14,189 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+describe("objectValInterpolation", () => {
+  it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized.", () => {
+    const people = [
+      { name: "ford prefect", occupation: "a hitchhiker" },
+      { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+      { name: "arthur dent", occupation: "a radio employee" },
+    ];
+    // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+
+    expect(objectValInterpolation(people)).toEqual([
+      "Ford Prefect is a hitchhiker.",
+      "Zaphod Beeblebrox is president of the galaxy.",
+      "Arthur Dent is a radio employee.",
+    ]);
+  });
+});
+
+// GOOD FAIL: ReferenceError: objectValInterpolation is not defined
 
 // b) Create the function that makes the test pass.
 
+/**
+--------- FUNCTION INFORMATION:
+Describe what the function will do: Function will take in an array of objects and return an array with a sentence about each person with their name capitalized.
+Function Name: objectValInterpolation
+Function Parameter(s): 1, array
+--------- INPUT / OUTPUT:
+input 1: array containing multiple objects
+output 1: an array using string interpolation to make the object key:value pair into a sentence, with the first letter of the element (value) capitalized
+ */
+
+// -------------------------------------------- INITIAL ATTEMPTS
+// -----------------------------------
+// I did a lot of practicing and testing with this in order to access the nested values. I knew what I needed to do to both access the data and capitalize it, but I started to overthink it once I attempted to create a function that would capitalize the word inside of the original function. After this, I decided to step back for a minute and then take another look at it, which resulted in the code underneath this one. I initially thought you couldn't use logic inside of string interpolation, and I knew that you COULD call a function, but it slipped my mind that HOF's are........ Well. They're functions. 😅 (It also turns out that I was probably wrong about using logic inside a template literal, my research on that has been a bit shaky.)
```

Comment:
You could set a variable of a name to the logic that you placed inside the template literal, but overall excellent work on this problem!